content/general/concepts/white-spaces/whitespace.md

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
The Docs folder (currently `white-spaces`) and file name (`whitespace.md`) under must be the same. Let's rename the folder to `whitespace`.